.github/workflows/support-frontend-build.yml

```diff
@@ -7,6 +7,7 @@ on:
   push:
     branches:
       - main
+      - rb-dummy-main
```

Comment:
do we want this?

Reply:
Argh! Well done you with the prize for spotting the deliberate mistake!